steamclog/src/main/java/com/steamclock/steamclog/Destinations.kt

```diff
@@ -105,9 +104,13 @@ internal class ConsoleDestination: Timber.DebugTree() {
  */
 internal class ExternalLogFileDestination : Timber.DebugTree() {
     private var fileNamePrefix: String = "sclog"
-    private var fileNameTimestamp = "yyyy_MM_dd"
     private var fileExt = "txt"
 
+    private var rotatingIndexes = (0 until 10).map { it }
```

Comment:
Not sure if it's any better or worse, but you could also initialize a List with an initializer function instead of running through a map:
`var rotatingIndexes = List(10) { it }`

This should also give you the List: `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]`